app/src/main/java/com/odoo/ContactDetailActivity.java

```diff
@@ -1,13 +1,626 @@
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.odoo.orm.ListRow;
+import com.odoo.table.ResCountry;
+import com.odoo.table.ResPartner;
+import com.odoo.table.ResState;
+import com.odoo.utils.BitmapUtils;
+
+import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
+import java.util.List;
 
-public class ContactDetailActivity extends AppCompatActivity {
+public class ContactDetailActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private ResPartner resPartner;
+    private ResState resState;
+    private ResCountry resCountry;
+    private Toolbar toolbar;
+    private CollapsingToolbarLayout collapsingToolbarLayout;
+    private FloatingActionButton fabEdit;
+    private TextView textMobileNumber, textPhoneNumber, textEmail, textStreet, textStreet2,
+            textCity, textState, textCountry, textPincode, textWebsite, textFax;
+    private ImageView profileImage, callImage;
+    private LinearLayout contactNumberLayout, emailLayout, addressLayout, websiteLayout, faxLayout;
+    private RelativeLayout mobileLayout, phoneLayout;
+    private LinearLayout editAddressLayout, editContactLayout, editEmailLayout, editWebsiteLayout,
+            editFaxLayout;
+    private EditText editMobileNumber, editPhoneNumber, editCity, editEmail, editState, editCountry,
+            editPincode, editWebsite, editFax, editStreet, editStreet2;
+    private String stringName, stringMobileNumber, stringPhoneNumber, stringEmail, stringStreet, stringStreet2,
+            stringCity, stringPincode, stringStateId, stringStateName, stringCountryId,
+            stringCountryName, stringWebsite, stringFax, stringImage;
+    private int _id;
+    private String address;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.contact_detail_activity);
+
+        toolbar = (Toolbar) findViewById(R.id.profile_toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setDisplayShowHomeEnabled(true);
+
+        collapsingToolbarLayout = (CollapsingToolbarLayout) findViewById(R.id.profile_collapsing);
+
+        fabEdit = (FloatingActionButton) findViewById(R.id.fabEdit);
+        fabEdit.setOnClickListener(this);
+
+        resState = new ResState(this);
+        resCountry = new ResCountry(this);
+
+        init();
+
+        _id = getIntent().getIntExtra("id", 0);
+        resPartner = new ResPartner(this);
+        List<ListRow> rows = resPartner.select("_id = ?", String.valueOf(_id));
+        if (rows != null) {
```

Comment:
rows never null. But it may be empty. Also you are selecting specific record so it will return only one record. So just select it by index 0 (zero)